proto/serverless/proto/instrumentation/tagset/v1/tagset.proto

```diff
@@ -0,0 +1,272 @@
+  string aws_service = 3;
+  // The name of the operation corresponding to the request.
+  string operation = 4;
+  // The unique ID of the request.
+  string request_id = 5;
+  // An optional error returned from the AWS APIs.
+  optional string error = 6;
+}
+
+message AwsSdkDynamodbTags {
+  // The AWS SDK base tags that all instrumented AWS SDK calls have.
+  AwsSdkBaseTags aws_sdk_tags = 1;
+  // The Dynamodb operation that was performed. Ex. GetItem, PutItem, Query, etc.
+  string operation = 2;
+  // The DynamoDB table name or names that the operation was performed on.
+  repeated string table_names = 3;
+  // The value of the ProjectionExpression request parameter.
+  optional string projection = 4;
+  // The value of the ScanIndexForward request parameter.
+  optional bool scan_forward = 5;
+  // The value of the AttributesToGet request parameter.
+  repeated string attributes_to_get = 6;
+  // The value of the ConsistentRead request parameter.
+  optional bool consistent_read = 7;
+  // The value of the IndexName request parameter.
+  optional string index_name = 8;
+  // The value of the Limit request parameter.
+  optional uint32 limit = 9;
+  // The value of the Select request parameter.
+  optional string select = 10;
+  // The value of the Segment request parameter.
+  optional uint32 segment = 11;
+  // The value of the TotalSegments request parameter.
+  optional uint64 total_segments = 12;
+  // The value of the Count response parameter.
+  optional uint64 count = 13;
+  // The value of the ScannedCount response parameter.
+  optional uint64 scanned_count = 14;
+}
+
+message AwsSdkSqsTags {
+  // The AWS SDK base tags that all instrumented AWS SDK calls have.
+  AwsSdkBaseTags aws_sdk_tags = 1;
+  // The SQS queue URL.
+  optional string queue_name = 2;
+  // The message IDs provided in the SDK operation response.
+  repeated string message_ids = 3;
+  // The SQS Operation that was performed.
+  optional string operation = 4;
+}
+
+message AwsSdkSnsTags {
+  // The AWS SDK base tags that all instrumented AWS SDK calls have.
+  AwsSdkBaseTags aws_sdk_tags = 1;
+  // The SNS Topic ARN, from the TopicArn request parameter.
+  optional string topic_name = 2;
+  // The SNS Operation that was performed.
+  optional string operation = 3;
+  // The message IDs provided in the SDK operation response.
+  repeated string message_ids = 4;
+}
+
+message NodeHttpTags {
+  // The URL of the HTTP request or response.
+  string url = 1;
+  // THe HTTP method of the request or response.
+  string method = 2;
+  // The HTTP path of the request or response.
+  string path = 3;
+  // The status code of the response.
+  uint32 status_code = 4;
+}
+
+message ExpressTags {
+  // The HTTP method defined by the Express Route Handler.
+  string method = 1;
+  // The HTTP Path defined by the Express Route Handler.
+  string path = 2;
+  // The status code returned by the Express Route Handler.
+  uint32 status_code = 3;
+}
```

Comment:
Same with these two messages, is it intentional that they are not added to the main Tags message?